### ISSUE TITLE
UserDetails Missing Accessible Feedback on Content Transition

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#796]
When the “I'm a Teacher” button Is activated, the page transitions to a form allowing teachers to input details about their role.  No accessible feedback is given to screen reader users to indicate that something has happened and that they should explore the page to locate the newly displayed form fields. 

Suggested Solution 
 
After following the advice in the previous “Missing Headings” section of this document, the first question on the teacher details form should be a level 2 heading.  Add a tabindex attribute to that heading, with a value of “-1”, and then move focus to it when the form content is loaded.  That is, once the user activates the “I'm a Teacher” button, the keyboard focus movement should cause them to hear something like: “What grade do you work with? Heading level 2”.

Aha! Link: https://diagramlabs.aha.io/features/B-122

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/8987a64612ed776e6c20bffed5e55b942ef528f9.
